pkg/wallet/api.go

```diff
@@ -143,3 +116,34 @@ func WithServiceType(serviceType string) DocOpts {
 
 // ErrKeyNotFound is returned when key not found
 var ErrKeyNotFound = errors.New("key not found")
+
+// ErrInvalidKey is used when a key is invalid
+var ErrInvalidKey = errors.New("invalid key")
+
+// errEmptyRecipients is used when recipients list is empty
+var errEmptyRecipients = errors.New("empty recipients")
+
+// errInvalidKeypair is used when a keypair is invalid
+var errInvalidKeypair = errors.New("invalid keypair")
+
+// VerifyKeys is a utility function that verifies if sender key pair and recipients keys are valid (not empty)
+func VerifyKeys(sender KeyPair, recipients [][]byte) error {
```

Comment:
Should utility function and non-exported error definitions be in api.go?

Reply:
good point, I can move them to utils.go

Reply:
done